src/ArachneTest.cc

```diff
@@ -639,12 +636,13 @@ TEST_F(ArachneTest, parseOptions_longOptions) {
     waitForTermination();
 
     int originalStackSize = stackSize;
-    int argc = 7;
+    int argc = 9;
     const char* argv[] =
         {"ArachneTest", "--minNumCores", "5", "--stackSize", "4096",
-            "--maxNumCores", "6"};
+            "--maxNumCores", "6", "--enableArbiter", "0"};
```

Comment:
As mentioned earlier, I believe we want to allow three options unless you have a strong reason not to.